authors/urls.py

```diff
@@ -27,4 +26,6 @@
     url(r'^api/', include('authors.apps.comments.urls')),
     url(r'^api/', include('authors.apps.profiles.urls')),
     url(r'^api/', include('authors.apps.bookmarks.urls')),
+    url(r'^api/', include('authors.apps.profiles.urls')),
```

Comment:
Kindly remove 1 `url(r'^api/', include('authors.apps.profiles.urls'))` profile URL namespace there are 2 in the urlpatterns

Reply:
I have made the changes